bundle/pom.xml

```diff
@@ -573,6 +573,16 @@
             <version>1.0.0</version>
             <scope>provided</scope>
         </dependency>
+        
+        <!--  We need to have this package on the build-path to avoid bnd complaining about
+              missing annotations (in the uber.jar these annotations are referenced)     
```

Comment:
maybe it should be clarified here, that this is only necessary due to the option `<_dsannotations-options>inherit</_dsannotations-options>` and the fact that we have OSGi components which inherit from legacy Felix SCR annotated classes.